projects/1.19/assets/dimensional-doors/dimdoors/lang/zh_cn.json

```
@@ -0,0 +1,426 @@
+  "commands.pocket.log.creation.on": "已启用口袋维度创建日志",
+  "commands.pocket.log.creation.generating": "从'%s'模板生成口袋维度于位置 %s %s %s",
+
+  "rifts.unlinked1": "这条裂痕没有连接任何地方",
+  "rifts.unlinked2": "这条裂痕已经关闭",
+  "rifts.isLocked": "这条裂痕已经锁定",
+  "rifts.cantUnlock": "无法解锁此维度门！",
+  "rifts.unlocked": "解锁",
+  "rifts.locked": "锁定",
+  "rifts.destinations.escape.cannot_escape_limbo": "很好的尝试，但你必须找到永恒源质才能逃离维度监牢",
+  "rifts.destinations.escape.not_in_pocket_dim": "此方块只能用于逃离口袋维度或维度监牢",
+  "rifts.destinations.escape.did_not_use_rift": "你没有使用维度门或裂痕离开口袋维度，所以你坠入了维度监牢!",
+  "rifts.destinations.escape.rift_has_closed": "用于进入口袋维度的裂痕已经关闭，所以你坠入了维度监牢!",
+  "rifts.destinations.private_pocket_exit.did_not_use_rift": "你没有使用维度门或裂痕离开口袋维度，所以你坠入了维度监牢!",
+  "rifts.destinations.private_pocket_exit.rift_has_closed": "用于进入口袋维度的裂痕已经关闭，所以你坠入了维度监牢!",
+  "rifts.entrances.rift_too_close": "在裂痕旁边放置维度门是很危险的。潜行右键以无视此限制，或放置于裂痕核心来连接此裂痕。",
+  "rifts.entrances.cannot_be_placed_on_rift": "这种类型的门无法放置于裂痕上",
+
+  "tools.rift_miss": "你只能在裂痕的核心处使用此物品",
+  "tools.signature_blocked": "错误：绑定处已被方块阻挡",
+  "tools.target_became_block": "创建失败，绑定处已被方块阻挡",
+
+  "text.autoconfig.dimdoors.title": "维度门",
+  "text.autoconfig.dimdoors.category.general": "全局设置",
+  "text.autoconfig.dimdoors.option.general.depthSpreadFactor": "Y轴扩散范围",
+  "text.autoconfig.dimdoors.option.general.depthSpreadFactor.@Tooltip": "逃离口袋维度或维度监牢后Y轴的扩散范围, 单位为方块/深度。次元监牢的默认深度为50。",
+  "text.autoconfig.dimdoors.option.general.riftCloseSpeed": "裂痕闭合速度",
+  "text.autoconfig.dimdoors.option.general.riftCloseSpeed.@Tooltip": "在使用裂痕移除器之后裂痕闭合的速度，以每个游戏刻的裂痕大小为单位。",
+  "text.autoconfig.dimdoors.option.general.riftGrowthSpeed": "裂痕成长速度",
+  "text.autoconfig.dimdoors.option.general.riftGrowthSpeed.@Tooltip": "裂痕扩大的速度，以每个游戏刻的裂痕大小为单位。",
+  "text.autoconfig.dimdoors.option.general.teleportOffset": "传送修正",
+  "text.autoconfig.dimdoors.option.general.teleportOffset.@Tooltip": "传送后玩家距离次元门的距离（以方块为单位）。",
+  "text.autoconfig.dimdoors.option.general.riftBoundingBoxInCreative": "创造模式裂痕碰撞体积",
+  "text.autoconfig.dimdoors.option.general.riftBoundingBoxInCreative.@Tooltip": "启用时，在创造模式将显示裂痕方块的碰撞体积。",
+  "text.autoconfig.dimdoors.option.general.endermanSpawnChance": "末影人生成概率",
+  "text.autoconfig.dimdoors.option.general.endermanSpawnChance.@Tooltip": "末影人生成于独立裂痕的概率。",
+  "text.autoconfig.dimdoors.option.general.endermanAggressiveChance": "末影人激怒概率",
+  "text.autoconfig.dimdoors.option.general.endermanAggressiveChance.@Tooltip": "生成于独立裂痕的末影人攻击最近玩家的概率。",
+
+  "text.autoconfig.dimdoors.category.doors": "维度门设置",
+  "text.autoconfig.dimdoors.option.doors.closeDoorBehind": "自动关闭维度门",
+  "text.autoconfig.dimdoors.option.doors.closeDoorBehind.@Tooltip": "启用时，在玩家进入维度门后它将自动关闭。",
+  "text.autoconfig.dimdoors.option.doors.doorList": "维度门",
+  "text.autoconfig.dimdoors.option.doors.doorList.@Tooltip": "调整某些维度门的属性。",
+  "text.autoconfig.dimdoors.option.doors.doorList.mode": "模式",
+  "text.autoconfig.dimdoors.option.doors.doorList.mode.@Tooltip": "启用：仅生成这些维度门对应的变种维度，禁用：禁止生成这些维度门对应的变种维度。",
```

上文为`启用时`，此处为`启用`，请保持统一
此外，启用/禁用之间的句子由于是并列关系，应该用分号割开
```suggestion
  "text.autoconfig.dimdoors.option.doors.doorList.mode.@Tooltip": "启用：仅生成这些维度门对应的变种维度；禁用：禁止生成这些维度门对应的变种维度。",
```